README.md

```diff
@@ -1,77 +1,131 @@
+
+*[Hello_Microverse]* is a HTML and CSS project
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- HTML
+- CSS
+- Linters
+
+### Key Features <a name="key-features"></a>
+
+- *Understanding Git, GitHub and GitHub Workflow*
+- *Creating pull requests*
+- *Configuring Linters*
+
+## 🚀 Live Demo <a name="live-demo"></a>
+- [Live Demo Link](https://nalianachei2020.github.io/Hello_Microverse)
+
+## 💻 Getting Started <a name="getting-started"></a>
+
 To get a local copy up and running, follow these steps.
 
 ### Prerequisites
+
 In order to run this project you need:
 
+- Visual Studio Code
+- Live server Extension
+
 ### Setup
-- Clone this repository to your desired folder on your local machine
+
+Clone this repository to your desired folder:
+sh
+- Create a folder
+- cd in to folder you just created
+- Type git clone https://github.com/NalianAchei2020/Hello_Microverse
 
 ### Install
+
+Install this project with:
+- cmd or vscode terminal
 - Install node js if you have not install
 - Go to your terminal and cd to folder, type npm install. This will install all the npm dependencies
 
 ### Usage
--  Run the project on chrome or any browser of your choice
+
+To run the project, start the html file on live server
 
 ### Run tests
+
 On your terminal, cd to the current folder and run 
 - **npx hint** to test  the html code
 - npx stylelint "**/*.{css,scss}" to test the css code
 
 ### Deployment
-You can create you repository and deploy this project on githut using:
-- This format for README file
-- Commit messages
-- Create a branch
-- Create a pull request
-
-### Authors
-Nalian Achei 
+
+You can deploy this project using:
+
+ - gh-pages
+ - netlify
+ - vercel
+
+## 👥 Authors <a name="authors"></a>
+
+👤 *Nalian Achei*
 - Github: [@NalianAchei2020](https://github.com/NalianAchei2020)
 - Linkedin: [Nalian Achei](https://www.linkedin.com/in/nalian-achei-683208275)
 - Twitter: [@NalianAchei](https://twitter.com/NalianAchei?t=E3ePLcJ7B45dBa8SBFIXDg&s=09)
 
-###  Contributing
-Contributions, issues, and feature requests are welcome!
+ ## 🔭 Future Features <a name="future-features"></a>
 
+- integration with javascript
+- Adding more html contents
+
+## 🤝 Contributing <a name="contributing"></a>
 Feel free to check the [issues page](https://github.com/NalianAchei2020/Hello_Microverse/issues).
-### Show your support
+
+## ⭐️ Show your support <a name="support"></a>
+
 If you like this project, please do well to give a star
 
-### Acknowledgments
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/NalianAchei2020/Hello_Microverse/issues).
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
 I would like to thank the microverse community for their lessons
 
-### Lincense
+## 📝 License <a name="license"></a>
+
 This project is [MIT](https://github.com/NalianAchei2020/Hello_Microverse/pull/2) licensed
+
```

Comment:
- [ ] [OPTIONAL]  The link to your `MIT LICENCE` is incorrect. Currently its just directing to a link to a pull request. Please use these [guidelines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to correctly add LICENCE into your README.